pkg/cli/project.go

```diff
@@ -88,6 +88,13 @@ func (a *Project) Run(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
+
+	for _, project := range projectDetails {
+		if project.Err != nil {
```

Comment:
If we failed to get project details, is it possible that `project.Project == nil`? If so, it looks like the for-loop below can still hit a nil pointer:

```go
for _, projectItem := range projectDetails {
		supportedRegions := projectItem.Project.GetSupportedRegions()
		defaultRegion := projectItem.Project.GetRegion()
		for i, supportedRegion := range supportedRegions {
			if supportedRegion == defaultRegion {
				supportedRegions[i] = supportedRegion + "*"
			}
		}
		out.Write(projectEntry{
			Name:    projectItem.FullName,
			Default: defaultProject == projectItem.FullName,
			Regions: supportedRegions,
		})
}
```

maybe we should add some checks before trying to access it there?

Reply:
good point, I consolidated the err check and now check for nil projectItem in this loop